composer.json

```diff
@@ -14,6 +14,7 @@
 	},
 	"require": {
 		"php": "~7.0",
+		"ext-xmlwriter": "*",
```

Comment:
I don't want PHPStan to depend on an extension, surely not for an optional feature like a error formatter. Can you rewrite the formatter to just concatenate strings?